### PR TITLE
Mod README, add a general policy for accepting/rejecting new features.

### PR DESCRIPTION
**Pull request by nowsprinting**
### Changes

- Fix wording
- Add a general policy for accepting/rejecting new features.

The general policy is just my idea.
If necessary, I would like to discuss.

---

### Contribution License Agreement

- [x] By placing an "x" in the box, I hereby understand, accept and agree to be bound by the terms and conditions of the [Contribution License Agreement](https://dena.github.io/cla/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -440,21 +440,30 @@ The instance of this Reporter (.asset file) can have the following settings.
...
-### Slack Reporter
+### SlackReporter
 
 A Reporter that post report to Slack.
 
 The instance of this Reporter (.asset file) can have the following settings.
 
 <dl>
-  <dt>Slack Token</dt><dd>Web API token used for Slack notifications. If omitted, no notifications will be sent.
+  <dt>Slack Token</dt><dd>Oath token of Slack Bot used for notifications. If omitted, no notifications will be sent.
```
Comment (get-me-power) [inline README.md:450]: I think thai it is OAuth token

Comment (nowsprinting) [reply README.md]: Thanks! fixed in f45f0226706a5483a3bd1217ac830a725b841bd0

**Review by get-me-power — APPROVED** [review]
LGTM

**Comment by get-me-power** [comment]
@nowsprinting 

Thanks for your contribution.

**Pull request merged**